From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Tue, 24 Jun 2025 11:28:00 +0000
Subject: [PATCH] guard login db deprecation for android feature

This is a temporary commit which will guard for potential backports of
enabling login databse deprecation for Android feature.
---
 .../password_manager/core/browser/features/password_features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/password_manager/core/browser/features/password_features.cc b/components/password_manager/core/browser/features/password_features.cc
index 91b902643a9aa..9360bf93d373b 100644
--- a/components/password_manager/core/browser/features/password_features.cc
+++ b/components/password_manager/core/browser/features/password_features.cc
@@ -164,7 +164,7 @@ BASE_FEATURE(kBiometricAuthIdentityCheck,
 
 BASE_FEATURE(kLoginDbDeprecationAndroid,
              "LoginDbDeprecationAndroid",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_DISABLED_BY_DEFAULT); // guard for potential backports
 #endif  // BUILDFLAG(IS_ANDROID)
 
 BASE_FEATURE(kUseNewEncryptionMethod,
